src/components/buttons/PopoverButton/PopoverButton.tsx

```diff
@@ -11,23 +12,43 @@ export interface PopoverButtonProps {
 
   /** Allows for additional cleanup when popover closes */
   onClose?: () => void;
+
+  /** State setter used in Select component to set focus when popover opens */
+  setIsPopoverOpen?: SetStateAction<any>;
```

Comment:
It seems like this could be

```suggestion
  setIsPopoverOpen?: (isOpen: boolean) => void;
```